book/graphs/other/mng01.qmd

```diff
@@ -228,7 +228,7 @@ webr_code_labels <- c("plot7")
 ::: {.panel-tabset .nav-justified}
 ## {{< fa regular file-lines fa-sm fa-fw >}} Preview
 
-```{r teal, opts.label = c("skip_if_testing", "app")}
+```{r teal, opts.label = c("skip_if_testing", "app"), eval = packageVersion("teal.modules.clinical") > "0.9.1"}
```

Comment:
teal.modules.clinical * 0.9.1.9031  2024-11-20 [1] https://p~

this was already working anyway